src/memote/suite/collect.py

```diff
@@ -168,3 +168,12 @@ def model(self):
     def sbml_version(self):
         """Provide SBML level, version, and FBC use."""
         return self._sbml_ver
+
+    def pytest_configure(self, config):
+        """Register custom markers at runtime."""
+        config.addinivalue_line("markers",
+                                "biomass: ")
```

Comment:
Usually markers are only the word itself, are you sure this is how it should be?

```suggestion
                                "biomass")
```

Reply:
It's possible that it is not required, however, all examples I found were in the format:
```
  def pytest_configure(self, config):
        """Register custom markers at runtime."""
        config.addinivalue_line("markers",
                                "marker: description of marker")
```

I'm happy to try it without the colon.